Python/Product/TestAdapter.Executor/Config/PythonProjectSettings.cs

```diff
@@ -83,31 +84,12 @@ public class PythonProjectSettings : IEquatable<PythonProjectSettings> {
                 UseLegacyDebugger == other.UseLegacyDebugger &&
                 TestFramwork == other.TestFramwork &&
                 UnitTestPattern == other.UnitTestPattern &&
-                UnitTestRootDir == other.UnitTestRootDir) {
-                if (SearchPath.Count == other.SearchPath.Count &&
```

Comment:
these collection values shouldn't matter.  projectName and projectHome should be all we need